### DISCUSSION TITLE
Double tap caps_word for capslock

### DISCUSSION THREAD

**Discussion by dcolestock** [manna-harbour/miryoku#187] [Q&A] [answered]
I added a feature to my layout that gives me caps lock if I double tap caps_word, just thought I'd share.

```
/ {
  behaviors {
    ct: capslocktap {
      compatible = "zmk,behavior-tap-dance";
      label = "CapsTap";
      #binding-cells = <0>;
      tapping-term-ms = <U_TAPPING_TERM_FAST>;
      bindings = <&u_caps_word>, <&kp CAPSLOCK>;
    };
  };
};

```

**Comment by manna-harbour** [comment, marked as answer]
Nice! By default there is Caps Lock if you combine Caps Word with Shift. You can hold Nav and Shift simultaneously. Do you prefer it with double tap instead?

**Reply by dcolestock** [reply]
Actually, I kinda like that better, didn't realize it was already in there, thanks!

**Reply by manna-harbour** [reply]
Great!

See https://github.com/manna-harbour/miryoku_zmk#caps-word.
